.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Can you run `git rm .vscode/launch.json` to remove this file on your branch, then do git add/commit/push?